Dockerfile

```
@@ -4,7 +4,7 @@ ARG DATOMIC_REPO_USER
 ARG DATOMIC_REPO_PASS
 ARG DATOMIC_LICENSE
 
-ENV DATOMIC_VERSION 0.9.6021
+ENV DATOMIC_VERSION 1.0.6202.0
```

```suggestion
ENV DATOMIC_VERSION 1.0.6202
```

As mentioned this one actually doesn't need the 4th digit 😅

---

Then i think we should change the script to also make the script like this, right?

---

Ah, right. You could use `sed` to strip away the 4th digit from the version number when setting this value, so `bump.sh 1.0.6202.0` just works.

---

See newest commit 👌